examples/cloudflare-workers/package.json

```diff
@@ -5,6 +5,7 @@
   "type": "module",
   "scripts": {
     "dev": "wrangler dev",
+    "clean": "rm -rf dist node_modules",
```

Comment:
Should we get rid of these clean scripts? They only work on windows and I don't know why we need them


Instead, we should have a `clean` file in the root that does this if/when it's needed

Reply:
Could absolutely remove them if you don't like them. I like having clean scripts if I want to test something from a clean sleight.

Reply:
I like having them too but the only example I really care about being representative of a "production environment" is the `next-prisma-starter`, the otherwise shouldn't contain unnecessary noise

Reply:
Alright, I'll remove them